rna/single_analysis.py

```diff
@@ -0,0 +1,282 @@
+"""
```

Comment:
willen we single_analysis.py houden? Lijkt extra code om te onderhouden zonder duidelijk doel?

Reply:
Die kan inderdaad weg, aangezien de nfold_analysis.py hetzelfde gaat doen.